### PR TITLE
Add preprocess to the adapter-static code example

### PR DESCRIPTION
**Pull request by stiegi**
Without preprocess you get "unexpected token" errors when importing types, as described here: https://stackoverflow.com/questions/72846408/sveltekit-typescript-parse-error-when-importing-types/72846510#72846510

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

File: documentation/docs/25-build-and-deploy/50-adapter-static.md [hunk 1]
```diff
@@ -14,8 +14,10 @@ Install with `npm i -D @sveltejs/adapter-static`, then add the adapter to your `
 // @errors: 2307
 /// file: svelte.config.js
 import adapter from '@sveltejs/adapter-static';
+import preprocess from 'svelte-preprocess';
```
Comment (dummdidumm) [inline documentation/docs/25-build-and-deploy/50-adapter-static.md:17]: We probably should use `vitePreprocess` here to be in line with the starter template

**Comment by gtm-nayan** [comment]
Which example are you referring to specifically? I don't think we have any snippet that necessitates a preprocessor on that page.

**Comment by s3812497** [comment]
https://kit.svelte.dev/docs/adapter-static#usage
This is the page and section referred to.

**Comment by dummdidumm** [comment]
I think the problem is that people just copy this verbatim without knowing that they might to adjust things if they are using a preprocessor, and then getting confused by that.
On the other hand, if someone who chose to not use TypeScript at all then they have the reverse problem of `svelte-preprocess` not being installed, and also being confused. So we either have a short comment there instead like `// add preprocessor, if necessary (when you want to use TS, for example)`, or just leave it be.

**Comment by stiegi** [comment]
I think it would be good enough to have a comment that mentions the preprocess.
The problem with this is, for a newcomer, the error message you get will make no sense and it is hard to go from that type error message to the missing preprocessor in the config file. So a hint that specifically targets typescript users might help a lot for newcomers imo

**Comment by Rich-Harris** [comment]
Thanks for the PR but I'm against this change — as has been pointed out, it's just as likely that people will add an unwanted preprocessor as that they'll remove one they need. Adding something irrelevant to the code sample just obscures the bit we need to communicate.

If developers blindly copy and paste code, errors like this one will be the least of the problems caused!

We could argue that the error message itself should be more informative in this case, but that's a separate conversation

**Pull request closed without merging**